runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler.cpp

```diff
@@ -113,8 +113,13 @@ OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler_initCifNativeTh
 		goto done;
 	/* Only intended for strut as the primitive's ffi_type is non-null */
```

Comment:
While you're here, please fix this comment: strut -> struct, and end with a period.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/38245ec2872c2b2664f6fdd3947f80e71e63c57d#diff-7e388e6ac3235f9470ab3d7607ffbe1c8fceacddd69ac992fdae6cda3e99257d